webviz_subsurface/_models/inplace_volumes_model.py

```diff
@@ -93,6 +95,16 @@ def __init__(
                 self._dataframe, self.pmodel.sens_df, on=["ENSEMBLE", "REAL"]
             )
 
+        # create HC_TOTAL columns
+        if "STOIIP" in self._dataframe and "ASSOCIATEDOIL" in self._dataframe:
+            self._dataframe["STOIIP_TOTAL"] = self._dataframe["STOIIP"].fillna(
+                0
```

Comment:
is the `fillna(0)` split needed here?

Reply:
Yes it is due to e.g. the `STOIIP` being `nan` for `FLUID_ZONE != "oil"`.. so need to fill it before adding them together otherwise the resulting `STOIIP_TOTAL` is also `nan` 🙂